paul-karen/server.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 3000;
+
```

Comment:
You're missing the comment to explain what's happening with this public directory.